modules/mongodb/manifests/backup.pp

```diff
@@ -89,6 +89,7 @@
       service_description => $service_desc,
       freshness_threshold => $threshold_secs,
       host_name           => $::fqdn,
+      notes_url           => monitoring_docs_url(mongodb),
```

Comment:
`monitoring_docs_url` generates a link to the alerts docs, so this would link to https://docs.publishing.service.gov.uk/manual/alerts/mongodb.html which doesn't exist.

Is there a doc page for this specific alert that we can link to? That would be more useful than the general docs on MongoDB anyway.

Reply:
I can create one instead.

Reply:
@suzannehamilton Done here: https://github.com/alphagov/govuk-developer-docs/pull/735/files